src/test/java/org/opensearch/ad/TestHelpers.java

```diff
@@ -1001,6 +1084,44 @@ public static void createIndex(RestClient client, String indexName, HttpEntity d
             );
     }
 
+    public static void createIndexWithHCADFields(RestClient client, String indexName, Map<String, String> categoryFieldsAndTypes)
+        throws IOException {
+        StringBuilder indexMappings = new StringBuilder();
+        indexMappings.append("{\"properties\":{");
+        for (Map.Entry<String, String> entry : categoryFieldsAndTypes.entrySet()) {
+            indexMappings.append("\"" + entry.getKey() + "\":{\"type\":\"" + entry.getValue() + "\"},");
+        }
+        indexMappings.append("\"timestamp\":{\"type\":\"date\"}");
+        indexMappings.append("}}");
+        createEmptyIndex(client, indexName);
+        createIndexMapping(client, indexName, TestHelpers.toHttpEntity(indexMappings.toString()));
+    }
+
+    public static void createAnomalyResultIndex(RestClient client) throws IOException {
```

Comment:
nit: name change to `createEmptyAnomalyResultIndex` or `createAnomalyResultIndexWithNoData`

Reply:
Sure, changed to first suggestion